### PR TITLE
Added VertexAttrib functions.

### PR DESCRIPTION
**Pull request by aqrs**
Added a function related to VertexAttrib.
We hope you will merge them.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -759,6 +759,22 @@ pub trait HasContext {
...
 
+    unsafe fn vertex_attrib_1_f32(&self, index: u32, x: f32);
+
+    unsafe fn vertex_attrib_2_f32(&self, index: u32, x: f32, y: f32);
+
+    unsafe fn vertex_attrib_3_f32(&self, index: u32, x: f32, y: f32, z: f32);
+
+    unsafe fn vertex_attrib_4_f32(&self, index: u32, x: f32, y: f32, z: f32, w: f32);
+
+    unsafe fn vertex_attrib_1_f32_slice(&self, index: u32, v: &[f32; 1]);
```
Comment (grovesNL) [inline src/lib.rs:770]: Could we use `&[f32]` for `v` in these? It's usually slightly easier to use and would match `uniform_1_f32_slice`

Comment (aqrs) [reply src/lib.rs]: Thanks for the review.
I don't think it's a problem, so I've fixed it.

Comment (grovesNL) [reply src/lib.rs]: Thank you! Could we also make this change for `vertex_attrib_2_f32_slice`, `vertex_attrib_3_f32_slice`, and `vertex_attrib_4_f32_slice`?

Comment (aqrs) [reply src/lib.rs]: Thanks for the review.
Is this okay?

Comment (grovesNL) [reply src/lib.rs]: Yes this looks great, thank you!

**Pull request merged**